spec/lib/sim_keyboard_settings_spec.rb

```diff
@@ -0,0 +1,83 @@
+describe RunLoop::SimKeyboardSettings do
+  let(:keyboard_enabled) { Resources.shared.plist_with_software_keyboard(true) }
+  let(:keyboard_not_enabled) { Resources.shared.plist_with_software_keyboard(false) }
+  let(:empty_plist) { Resources.shared.empty_plist }
+
+  let(:physical) do
+    RunLoop::Device.new('name', '9.4',
+                        '30c4b52a41d0f6c64a44bd01ff2966f03105de1e')
+  end
+
+  let(:simulator) do
+    RunLoop::Device.new('iPhone 5s', '9.4',
+                        '77DA3AC3-EB3E-4B24-B899-4A20E315C318', 'Shutdown')
+  end
+
+  subject(:sim_keyboard) { RunLoop::SimKeyboardSettings.new(device) }
+
+  context 'software keyboard will show' do
+    let(:device) { simulator }
+
+    context '#soft_keyboard_will_show?' do
```

Comment:
Normally, this would be a describe, not a context. You are describing the specification of the #soft_keyboard_wil_show? instance method.